<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controle Localfrio-LAGES</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="abas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1901</width>
      <height>1061</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="TAB_Pedido">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Pedido</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="LBL_Pedido">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Número do Pedido</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="LBL_DL_Fabrica">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Deadline da Fábrica</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="LBL_InicioJanela">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Inicio da Janela</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="LBL_Fabrica">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Origem</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="LBL_Status">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="TXB_Pedido">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="DATE_DL_Fabrica">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDateEdit" name="DATE_InicioJanela">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="CBX_Fabrica">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Lages</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Otacílio Costa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Correia Pinto</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="CBX_Status">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Aberto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Em Coleta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Em Unitização</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Em Envio Porto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Finalizado</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="PBT_Novo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Novo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="LBL_Booking">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>Booking</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="LBL_DL_Porto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Deadline do Porto</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="LBL_FimJanela">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fim da Janela</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="LBL_Porto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Destino</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QRadioButton" name="RBT_Cabotagem">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cabotagem</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5" rowspan="2">
       <widget class="QPushButton" name="PBT_Savar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QLineEdit" name="TXB_Booking">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="1" rowspan="2">
       <widget class="QDateEdit" name="DATE_DL_Porto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="2" rowspan="2">
       <widget class="QDateEdit" name="DATE_FimJanela">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3" rowspan="2">
       <widget class="QComboBox" name="CBX_Porto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Itajaí</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Navegantes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Itapoá</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QRadioButton" name="RBT_Expurgo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Expurgo</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5" rowspan="2">
       <widget class="QPushButton" name="PBT_Buscar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" rowspan="2">
       <widget class="QLabel" name="LBL_Armador">
        <property name="text">
         <string>Armador</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" rowspan="2">
       <widget class="QLabel" name="LBL_TerminalVazio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Terminal Vazio</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" rowspan="2">
       <widget class="QLabel" name="LBL_QtdeCntr">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Quantidade</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5" rowspan="2">
       <widget class="QPushButton" name="PBT_Excluir">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QComboBox" name="CBX_Armador">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>HSUD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MSC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MAERSK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CMA CGM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HAPAG LLOYD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COSCO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MERCOSUL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ONE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ZIM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>YML</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FR MEYERS</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="CBX_TerminalVazio">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>ATM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rogério Phillip-ITJ</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSpinBox" name="SBX_QtdeCntr">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="6">
       <widget class="QTableWidget" name="TABLE_Status">
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Container</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="TAB_Estoque">
     <attribute name="title">
      <string>Estoque</string>
     </attribute>
     <widget class="QTableWidget" name="TABLE_Estoque">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>751</width>
        <height>411</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Booking</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CNTR</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>OBS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>FreeTime</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="LBL_BK">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Booking</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TXB_BK">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LBL_CNTR">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Container</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="TXB_CNTR">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LBL_StatusEstoque">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CBX_ArmadorEstoque">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>A Diposição</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Unitizado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Finalizado</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="PBT_AplicarFiltro">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Aplicar Filtros</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PBT_EditarCNTR">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Editar CNTR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PBT_IncluirCNTR">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Incluir</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TAB_Porto">
     <attribute name="title">
      <string>Carregamento</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="TAB_Veiculo">
     <attribute name="title">
      <string>Veículos</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
